test/test.js

```diff
@@ -1,9 +1,13 @@
-const assert = require('assert');
+
 const got = require('got');
-const server = require('../src/index.js');
+const server = require('../src');
```

Comment:
It's better to use just '../' to force Node for loading your app via `package.json`.